.github/workflows/performance.yml

```
@@ -0,0 +1,211 @@
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+      
+    - name: Setup Flutter
+      uses: subosito/flutter-action@v2
+      with:
+        flutter-version: '3.19.0'
+        channel: 'stable'
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)